### PR TITLE
Remove homepage step from Personalize task for Tsubaki theme

### PR DESCRIPTION
**Pull request by chihsuan**
### Changes proposed in this Pull Request:

Closes https://github.com/Automattic/wc-calypso-bridge/issues/1028.

This PR removes the homepage step from Personalize task for Tsubaki theme.

- [ ] This PR is a very minor change/addition and does not require testing instructions (if checked you can ignore/remove the next section).

### How to test the changes in this Pull Request:

- [ ] Have you followed the [Writing high-quality testing instructions guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions)?

1. Set up a free trial site
2. Go to WooCommerce > Home > Personalize task
3. Observe that the home page step is not shown.

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: includes/tasks/class-wc-calypso-task-appearance.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+class WCBridgeAppearance extends Appearance {
+	/**
+	 * Addtional data.
+	 *
+	 * @return array
+	 */
+	public function get_additional_data() {
+		return array(
+			'has_homepage' => self::has_homepage(),
+			'has_products' => HeadstartProducts::has_products(),
```
Comment (chihsuan) [inline includes/tasks/class-wc-calypso-task-appearance.php:23]: I use `HeadstartProducts` here so we can get the correct products count to exclude sample products generated by Headstart themes

**Comment by ilyasfoo** [comment]
This fix tested well, however, may have unintended effect. Now, import sample product is listed in the step instead. Is this ok @FlorianBru? 

image

**Review by ilyasfoo — APPROVED** [review]
Approving anyway since LGTM, we could fix later if needed. 🚢

**Comment by FlorianBru** [comment]
Hey @ilyasfoo, is there an option to test the described scenario?

Regarding 'import sample products', what do you think, how much time will it take to also remove this step?

**Comment by ilyasfoo** [comment]
Hi @FlorianBru, I've invited you as administrator to my testing site, https://freetrialtestingsite5.wpcomstaging.com/wp-admin/admin.php?page=wc-admin. You could try to go to "personalize your store" task to see it.

>how much time will it take to also remove this step?
The removal will be quite simple to achieve, but since we've already missed the deploy train today, deployment would be tomorrow the soonest, but next week more likely.

**Pull request merged**